app/templates/email/ride_approved.txt

```diff
@@ -2,7 +2,7 @@ Hi {{ rider['name'] }},
 
 Good news! {{ carpool['driver']['name'] }} approved your request to join the carpool from {{ carpool['from_place'] }} to {{ carpool['to_place'] }}, departing at {{ carpool['leave_time'] | humanize }}.
 
-Please contact {{ carpool['driver']['name'] }} at {{carpool.driver.email}} or DRIVER_PHONE to figure out exactly when and where to meet.
+Please contact {{ carpool['driver']['name'] }} at {{carpool.driver.email}} or {{carpool.driver.phone_number}} ({{carpool.driver.preferred_contact_method}} preferred) to figure out exactly when and where to meet.
```

Comment:
Gotta do dictionary notation here, not dot.

Reply:
Dot works. Happy to change if for style reasons if you like.

Reply:
Woah. mindblown.gif. Wonder why it wasn't working for me before.

Reply:
Ah hah! The reason we still need dict notation here is that these database objects currently have to be serialized to fit on the Redis queue. The in-browser preview we wrote in `/admin` will work because it doesn't have to cross the Redis barrier.

For now, let's keep going with dict notation here and we can fix this problem later by just sending the UUIDs across the wire and the email worker can do the database lookup and pass it in to the template.